src/casts/interpret-casts-common.rkt

```diff
@@ -174,55 +177,13 @@ TODO write unit tests
         (Type-MVect (interp-tglb (type-mvec-of$ t1) (type-mvec-of$ t2)))]
        [(and$ (type-tup?$ t1) (type-tup?$ t2)
               (op<=? (type-tup-arity$ t2) (type-tup-arity$ t1)))
-        (Make-GLB-Two-Tuple-Types tglb-uid t1 t2)]
+        (Make-GLB-Two-Tuple-Types types-greatest-lower-bound t1 t2)]
        [else (Error (Quote "inconsistent types"))])))
-  (: tglb : Schml-Type Schml-Type -> (Option Schml-Type))
-  (define (tglb t1 t2)
-    (: tglb* : Schml-Type* Schml-Type* -> (Option Schml-Type*))
```

Comment:
It is worth noting that I deleted this function because tglb is a runtime function only and there is no way it can run on statically-known types.